R/query_dataset.R

```diff
@@ -28,35 +28,45 @@
 #'
 #' ## geographies
 #'
-#' Geographies can be supplied as a vector or a data frame depending on the complexity of the
-#' desired query.
-#'
-#' A vector will run a query returning **any** rows meeting any of the given
-#' geographies, i.e. geographies = `c("NAT", "REG")` will return all national and regional level
-#' rows, whilst `c("NAT", "REG|code|E120000001")` will return all national level rows and all North
-#' East rows. Specific locations are required to be supplied in the format
-#' `"LEVEL|identifier_type|identifier"`, where identifier type can be either code or id and the
-#' corresponding identifier is then the standard ONS code or the sqid given in the meta data
-#' respectively. Using England as an example, these would be:
+#' Geographies can be supplied as a single string, a vector of strings, a list or a data frame
+#' depending on user preference and the complexity of the desired query.
+#'
+#' The user can supply a **single string** containing the desired geographic level (e.g. `"NAT"`,
+#' `"REG"`, `"LA"`) or a specific location (e.g. `"NAT|code|E92000001"`, `"REG|code|E12000001"`)
+#' and the query will search for rows at that single level or location.
+#'
+#' Specific locations are required to be supplied in the format
+#' `"location_level|location_id_type|location_id"`, where location_id_type can be either code or id
+#' and the corresponding location_id is then the standard ONS code or the sqid given in the meta
+#' data respectively. Using England as an example, these would be:
 #'   - `"NAT|code|E92000001"`
 #'   - `"NAT|id|dP0Zw"`
 #'
-#' If you require a more complex selection, for example all LAs in a given region, then a data
-#' frame should be supplied, with a row for each selection. Note however, that this will only work
-#' when using the default `POST` method. The `GET` method is much more limited and can not process
-#' more complex queries.
+#' Using a **vector** will run a query returning **any** rows meeting any of the given geographies,
+#' i.e. geographies = `c("NAT", "REG")` will return all national and regional level rows, whilst
+#' `c("NAT", "REG|code|E120000001")` will return all national level rows and all North
+#' East rows.
+#'
+#' If you require a more complex selection, for example all LAs in a given region, then a **data
+#' frame** should be supplied, with a row for each selection. Note however, that this will works
```

Comment:
that this will *work best (would have suggested changes but as you'll need to redocument afterwards anyway figured it wasn't worth it)

Reply:
Updated